TabulateSmarterTestAdminPackage.Common/Validators/RequiredDateTimeValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using TabulateSmarterTestAdminPackage.Common.Enums;
+
+namespace TabulateSmarterTestAdminPackage.Common.Validators
+{
+    public class RequiredDateTimeValidator : Validator
+    {
+        public RequiredDateTimeValidator(ErrorSeverity errorSeverity, object parameter) : base(errorSeverity, parameter)
+        {}
+
+        public override bool IsValid(object value)
+        {
+            DateTime dateTimeValue;
+            var valueString = value as string;
+            return DateTime.TryParse(valueString, out dateTimeValue);
```

Comment:
Up to you, but `return DateTime.TryParse(value as string, out dateTimeValue);` seems slightly cleaner